### DISCUSSION TITLE
Beta testers needed for hosted CRUD API generator

### DISCUSSION THREAD

**Discussion by polterguy** [polterguy/magic#186] [Announcements]
Hi community, we just launched our hosted Low-Code generator based upon Magic Cloud yesterday, and we're looking for BETA testers wanting to try out our things. Basically, our hosting solution allows you (for free) to signup in a minute, and generate your CRUD apps, without any download requirements, or anything else. Check it out below if this sounds appealing.

* [BETA test our CRUD generator](https://aista.com?secret=github)

**Comment by ldj670** [comment]
As a Structural Engineer in Denmark, I am constantly required to adhere to a vast array of rules and standards in my daily work. This can be both tedious and time-consuming, and I am eager to find ways to optimize this aspect of my job.

What I enjoy most about my profession is conducting structural analyses and evaluating the strength of building materials. This includes analyzing forces and designing structures based on the capacity of the materials.

However, a significant portion of my time is spent ensuring compliance with the various norms and regulations that apply to my work. These rules can vary depending on factors such as the type of building, its intended use, the materials used, its geographical location, and its architectural design. Furthermore, with the recent implementation of a single code for all of Europe, navigating these regulations has become even more challenging as each country still has its own set of national annexes.

In short, there are a vast number of rules and documents to review and it can be very time-consuming and tedious.

Given this, I am wondering if it would be possible to use "Magic Cloud " to train a chatbot that could assist me in answering questions related to these regulations, thus eliminating the need to spend so much time browsing through legal-like documents. These documents are readily available online but are often in PDF format and spread across multiple documents.

Would it be possible for this "Magic Cloud " to help make my life, and that of many other structural engineers in Europe, easier?"

**Reply by polterguy** [reply]
Hi there, and thank you for an intelligent question. It would be possible. The largest problem is to generate training data for your own model, but it should be possible to overcome somehow. I am currently focusing on implementing support for training models based upon websites, but training from PDF documents is an interesting thing, I would love to implement.

I'm not sure about any time frames for this, but if you contact me at th@aista.com, I would love to have a look at your particular requirements, to see if I can extract a common solution solving _"two problems in one go"_.
